server/client/js/main.js

```diff
@@ -45,13 +46,14 @@ firebase.auth().onAuthStateChanged(user => {
 
         // Retain reference to user doc in firestore
         userDoc = users.doc(user.uid);
+        lastOperationsCol = userDoc.collection("lastOperations");
 
-        // Get user doc info
-        userDoc.get()
-            .then(docSnapshot => {
-                let data = docSnapshot.data();
-                if (data) {
-                    lastOperation = data.lastOperation;
+        // Get lastOperations timestamps
+        lastOperationsCol.get()
+            .then(querySnapshot => {
+                for (let docSnapshot of querySnapshot.docs) {
+                    let data = docSnapshot.data();
+                    lastOperations[docSnapshot.id] = data.lastOperation;
```

Comment:
So from what I understand, each docSnapshot refers to a room and this gets the last operation. Awesome!